apps/vmq_server/priv/lua_scripts/enqueue_msg.lua

```diff
@@ -0,0 +1,19 @@
+#!lua name=enqueue_msg
+
+--[[
+ARGV[1] = Node
+ARGV[2] = SubscriberId
+ARGV[3] = SubInfo&Msg
+]]
+
+local function enqueue_msg(_KEYS, ARGV)
+    local node = ARGV[1]
+    local subscriberId = ARGV[2]
+    local msg = ARGV[3]
+
+    local mainQKey = 'mainQueue'.. '::'.. node
```

Comment:
Should `mainQueue` be taken as a var from _KEYS args?

Reply:
Any specific reason to do so?

Reply:
Fewer hardcoded values in script, as I also see that in other parts of code (poll_main_queue script I think), key is being passed as an arg but here it is hardcoded.